pages/common/cryfs.md

```diff
@@ -0,0 +1,20 @@
+# cryfs
+
+> A cryptographic filesystem for the cloud.
+> More information: <https://www.cryfs.org/>.
+
+- Mount an encrypted file system. The initialization wizard will be started on the first time:
```

Comment:
Perhaps this will read better?
```suggestion
- Mount an encrypted file system. The initialization wizard will be started on the first execution:
```